.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: scala
 scala:
-- 2.11.12
-- 2.12.5
+- 2.13.3
 jdk:
-- oraclejdk8
+- openjdk8
+services:
+- docker
 script:
-- sbt ++$TRAVIS_SCALA_VERSION ";core/test;docker/test;example/test;example/copyReadme"
+- docker pull subfuzion/netcat:latest
```

Comment:
Was unable to pull images as part of the sbt build via docker-client (resulted in 404s)
Shout if there's something in Travis blocking this that I don't know about please!